docs/build/sdks/swift/contracts.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
When adding new files, we need to always update the navigation tree. I'll take care of it for this PR